### ISSUE TITLE
Edit Permissions

### ISSUE THREAD

**Issue by GlennRicaud** [GlennRicaud/data-toolbox-app#10]
Original request from @ComLock
```
You can create repos using datatoolbox, but you cannot set rootPermission like the api supports:
http://repo.enonic.com/public/com/enonic/xp/docs/6.15.5/docs-6.15.5-libdoc.zip!/module-repo.html#.create
Also you cannot edit the permissions of the root node after you have created a repo.
It would be really nice to have either.
```
```
Do support this properly I guess one need a principal selector (role, group, user )
```

**Comment by sigdestad** [comment]
Glenn - any chance you could look into this issue?

**Comment by GlennRicaud** [comment]
Will prioritize this :+1:
Should be done by the ~~end of the week~~
EDIT: next week

**Comment by sigdestad** [comment]
Awesome :-)

**Comment by GlennRicaud** [comment]
Edit of permissions will be released with DT 5.0.17
